internal/action/bufpane.go

```diff
@@ -815,6 +815,7 @@ var BufKeyActions = map[string]BufKeyAction{
 	"ToggleRuler":               (*BufPane).ToggleRuler,
 	"ToggleHighlightSearch":     (*BufPane).ToggleHighlightSearch,
 	"UnhighlightSearch":         (*BufPane).UnhighlightSearch,
+	"ResetSearch":               (*BufPane).ResetSearch,
```

Comment:
Sorry for my late response.
As a last small change request: `ResetSearch` should be added to [keybindings.md#bindable-actions-and-bindable-keys](https://github.com/zyedidia/micro/blob/master/runtime/help/keybindings.md#bindable-actions-and-bindable-keys).

If this is added it can be merged. :+1:

Reply:
> Isn't there are better way to keep that part of `keybindings.md` in sync with the BufKeyActions map?

Currently I can only imagine to generate some parts of the docs via e.g. `make doc`.